### ISSUE TITLE
Отвезти 4 книги в хранилище библиотек Книжной Палаты

### ISSUE THREAD

**Issue by erulki** [MikStepv/MS#7]
Надо узнать у них куда отвезти эти книги - что важно: обязательно взять бумагу или подтверждение о предоставлении этих 16 экземпляров!

Российское национальное агентство ISBN находится по адресу: 
129085, Москва, Звездный бульвар, д. 17, строение 1, ИТАР-ТАСС, РКП, Административный корпус, офис 306
Телефон:
(495) 688 96 89,
(499) 791-01-10, доб.: 51-06, 51-14, 51-15.
Электронная почта: ISBN@tass.ru

Может быть в пятницу после меня ты сможешь отвезти книги на такси?
Узнай по времени?

**Comment by erulki** [comment]
так, беру в работу. какие книги нужно отвезти будет? что сказать им когда позвоню, как представиться?

**Comment by MikStepv** [comment]
Помощник Михаила Степанова

Книги завтра тебе дам. 

Узнай до какого часа они работают завтра? Чтобы успеть после нашей встречи?

**Comment by erulki** [comment]
я отправила письмо им на почту. если не получу ответ - наберу им завтра в течение дня

**Comment by erulki** [comment]
переслала вам на почту их ответ

**Comment by MikStepv** [comment]
Уточнить у них:

1. Есть ли сроки доставки книги им? - Ты можешь прочитать книгу сама и потом ее отдать в библиотеку
2. 16 книг это обязательно? Можно больше или меньше?
3. Можно но ли вообще не давать?

Идея, если нет срока - то спокойно потом.

**Comment by erulki** [comment]
позвоню им в понедельник, потому что они закрыты на выходных, не думаю, что кто-то мне ответит

**Comment by erulki** [comment]
1. в любой день, если своим ходом, до 16 часов. срока нет, можно привезти когда угодно, хоть через год, условно
2. 16 книг идут в 16 библиотек. если важно, чтобы они попали в самые главные библиотеки - можно привезти только 4
3. да, можно и вообще их не привозить. если хотим, чтобы книги были в каталоге библиотек и размещались в библиотеках - привозим, если нет - не привозим

вообще почти на любой мой вопрос ответ начинался на «ой, ну это вы смотрите сами»))
